src/client/components/TabbedContainer/index.js

```diff
@@ -1,9 +1,18 @@
+import React, { useState } from "react";
+import Button from "react-bootstrap/Button";
 import Col from "react-bootstrap/Col";
 import Container from "react-bootstrap/Container";
 import Nav from "react-bootstrap/Nav";
-import React from "react";
 import Row from "react-bootstrap/Row";
 import Tab from "react-bootstrap/Tab";
+// We generate these tab names dynamically and assign them to TabPaneContentTagName
+import TabPaneContent0 from "../TabPaneContent0"; // eslint-disable-line no-unused-vars
```

Comment:
Do you need the eslint-disable-line here? It looks like you're using the import below.

Reply:
great catch, you're right i don't need them anymore! still seems hacky, less hacky than before :P